packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -1082,6 +1082,17 @@ export class UserInputUtil {
         }
     }
 
+    public static async failedNetworkStart(popupMsg: string, logMsg: string, outputAdapter: VSCodeBlockchainOutputAdapter): Promise<void> {
```

Comment:
No need to pass outputAdapter through the function - you can just call:
`const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();`